fy22_cecil_b_moore/condition_sum.sql

```diff
@@ -0,0 +1,86 @@
+select 
```

Comment:
this file feel ripe for refactoring but I don't have a specific suggestion at this moment. Fine as-is as long as the outputs are correctly representative of the inputs.